vroom/main.vroom

```diff
@@ -18,7 +18,30 @@ stub that out for vroom.
   :endfunction
   :call maktaba#test#Override('repeat#set', 'FakeRepeat')
 
-We'll also stub it out to not care whether certain executables are installed on
+
+Before we stub out availability checks, test :FormatCode for unavailable plugins
```

Comment:
Hmm, putting this gnarliness at the very beginning of the file would undermine the sequence of concepts I'm trying to build in #11. Could we just add a `codefmt#ReenableIsAvailableChecksForTesting()` or `codefmt#OverrideIsAvailableChecksForTesting({is_enabled})` and put further down in the file where it's more natural?